rest/voice/outbound-calls/example-4/example-4.4.x.rb

```diff
@@ -8,10 +8,10 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 call = @client.account.calls.create(:url => "http://demo.twilio.com/docs/voice.xml",
-    :to => "+14155551212",
-    :from => "+18668675309",
-    :method => "GET",
-    :status_callback => "https://www.myapp.com/events",
-    :status_callback_method => "POST",
-    :status_callback_event => ["initiated", "ringing", "answered", "completed"])
+  :to => "+14155551212",
+  :from => "+18668675309",
+  :method => "GET",
+  :status_callback => "https://www.myapp.com/events",
+  :status_callback_method => "POST",
+  :status_callback_event => ["initiated", "ringing", "answered", "completed"])
```

Comment:
In situations like this you can place the first parameter right below the open paren, e.g.

```ruby
call = @client.account.calls.create(
  :url => "http://demo.twilio.com/docs/voice.xml",
  # ...
```